runhouse/rns/hardware/cluster.py

```diff
@@ -628,8 +669,15 @@ def run_python(
         return return_codes
 
     def send_secrets(self, providers: Optional[List[str]] = None):
```

Comment:
would `get/sync/retrieve_secrets` make more sense? `cluster.send_secrets` makes it sound like the secrets on the cluster are being sent, rather than sending from local to the cluster.